scripts/fuzz/generate_decoding_fuzz_corpus.py

```
@@ -0,0 +1,91 @@
+        str(phase14_path),
+    )
+
+    phase12 = json.loads(phase12_path.read_text())
+    write_json(phase12_path, phase12)
+
+    phase14 = json.loads(phase14_path.read_text())
+    write_json(phase14_path, phase14)
+
+    lookup_artifacts = phase12.get("shared_lookup_artifacts", [])
+    if not lookup_artifacts:
+        raise SystemExit("phase12 corpus generation did not produce shared lookup artifacts")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)